packages/parse5/lib/serializer/index.test.ts

```diff
@@ -42,4 +42,15 @@ describe('serializer', () => {
 
         assert.equal(html, '<button>Hello</button>');
     });
+
+    it('serializes the children of void elements as the empty string (GH-289)', () => {
+        const document = parse5.parseFragment('<br/>');
```

Comment:
```suggestion
        const document = parse5.parseFragment('<br>');
```

IMO using the self-closing slash here gives off the idea that ***that*** is what makes an element “void”. It’s not, the slash is an XML-ism that does not do anything in HTML. Void elements are elements with a certain known tag name.

Reply:
Makes sense, happy to update